stats/gRPC.md

```diff
@@ -0,0 +1,125 @@
+* Dimensionless values have unit "1"
+
+Buckets for distributions in default views are as follows:
+
+* Size in bytes: 0, 1024, 2048, 4096, 16384, 65536, 262144, 1048576, 4194304, 16777216, 67108864, 268435456, 1073741824, 4294967296
+* Latency in ms: 0, 0.01, 0.05, 0.1, 0.3, 0.6, 0.8, 1, 2, 3, 4, 5, 6, 8, 10, 13, 16, 20, 25, 30, 40, 50, 65, 80, 100, 130, 160, 200, 250, 300, 400, 500, 650, 800, 1000, 2000, 5000, 10000, 20000, 50000, 100000
+* Counts (no unit): 0, 1, 2, 4, 8, 16, 32, 64, 128, 256, 512, 1024, 2048, 4096, 8192, 16384, 32768, 65536
+
+## Client
+
+### Measures
+
+Client stats are recorded for each individual gRPC request. All measurements except `grpc.io/client/started_count` must be recorded at the end of each gRPC request.
+
+| Measure name                               | Unit | Description                                                                                   |
+|--------------------------------------------|------|-----------------------------------------------------------------------------------------------|
+| grpc.io/client/started_count               |      | Number of all client requests started. Records "1" at the beginning of each gRPC request.     |
+| grpc.io/client/request_count               |      | Number of request messages in a gRPC request.                                                 |
+| grpc.io/client/request_bytes               | By   | Total compressed bytes sent in a gRPC request.                                                |
+| grpc.io/client/uncompressed_request_bytes  | By   | Total uncompressed bytes sent in a gRPC request.                                              |
+| grpc.io/client/response_count              |      | Number of response messages in a gRPC request.                                                |
+| grpc.io/client/response_bytes              | By   | Total compressed bytes received in a gRPC request.                                            |
+| grpc.io/client/uncompressed_response_bytes | By   | Total uncompressed bytes received in a gRPC request.                                          |
+| grpc.io/client/latency                     | ms   | Time between first byte of request received to last byte of response sent, or terminal error. |
+| grpc.io/client/server_latency              | ms   | Propagated from the server and should have the same value as "grpc.io/server/latency".        |
+
+### Tags
+
+All client metrics should be tagged with the following. Some tags are not available for measurements recorded at the beginning of the gRPC request (e.g. `grpc.canonical_status`).
+
+| Tag name              | Description                                                   |
+|-----------------------|---------------------------------------------------------------|
+| grpc.client_method    | Full gRPC method name, including package, service and method. |
+| grpc.canonical_status | gRPC canonical status code.                                   |
```

Comment:
I don't think we need the word "canonical". There is only one status for gRPC and so it's not ambiguous enough to require it in the key. We might want to call it "client_status" in case we want to separately record "server_status" / like you did with method.

Reply:
sure. done.